decidim-verifications/app/controllers/decidim/verifications/id_documents/admin/pending_authorizations_controller.rb

```diff
@@ -22,7 +22,7 @@ def pending_online_authorizations
               .new(organization: current_organization, name: "id_documents", granted: false)
               .query
               .where("verification_metadata->'rejected' IS NULL")
```

Comment:
How is this working if verification_metadata is encrypted?

Reply:
NULL values are not encrypted. If the JSON field is empty, it will appear as NULL in the database.

Reply:
ah, ok, my fault, it's only the values that are encrypted, not the whole hash itself, so the keys are still readable and "queryable".

Reply:
Yes, kind of "queryable".

The encrypted values cannot be queried directly, though, because I think the encrypted value is different on each call to `ActiveSupport::MessageEncryptor` which is what the `Decidim::AttributeEncryptor` class uses.